handlers/create_lease.go

```diff
@@ -25,7 +26,7 @@ func (c createLeaseReq) maxTimeDur() time.Duration {
 }
 
 // CreateLease creates the handler that responds to the POST /lease endpoint
-func CreateLease(containerService *container.Service) http.Handler {
+func CreateLease(containerService *container.Service, services k8s.ServiceInterface) http.Handler {
```

Comment:
This handler will need to use the `container.Service` to list the clusters in GKE, and the `k8s.ServiceInterface` to get annotations on the `k8s-claimer` service (which it will use to track lease ownership). See the [leasing algorithm design doc](https://github.com/deis/k8s-claimer/blob/master/doc/architecture.md#leasing-algorithm) for more details